ansible/roles/de-services-cfg/README.md

```diff
@@ -22,9 +22,8 @@ N/A
 Dependencies
 ------------
 
-util-cfg-logging
```

Comment:
It appears that util-cfg-logging isn't referenced at all anymore, although it could potentially be used to create local logging config files. Would it be reasonable to remove the role or is there a reason to keep it around?